spring-cloud-dataflow-docs/src/main/asciidoc/streams-skipper.adoc

```diff
@@ -96,6 +96,240 @@ $ curl http://localhost:9002/info
 {"app":{"description":"Spring Cloud Stream Log Sink Rabbit Binder Application","name":"log-sink-rabbit","version":"1.2.0.RELEASE"}}
 ----
 
+=== Force update of a Stream
+
+When upgrading a stream, the --force option can be used to deploy new instances of currently deployed applications even if no applicaton or deployment properties have changed.
```

Comment:
Looks like the stream `upgrade` and `update` are used interchangeably? Perhaps we should stick with only one? Seems like the `update` is prevailing in the rest of the doc.

Reply:
Yeah, we use `update` in the SCDF side while `upgrade` in the Skipper side. But, in some cases, the `update` fits for the cases of `upgrade` and `rollback` as well.

In this specific case, the `--force` option is for the `upgrading`